Northstar.CustomServers/mod/scripts/vscripts/mp/spawn.nut

```diff
@@ -1,27 +1,30 @@
 untyped
 
-global function InitRatings // temp for testing
-
 global function Spawn_Init
-global function SetRespawnsEnabled
-global function RespawnsEnabled
+global function FindSpawnPoint
+
 global function SetSpawnpointGamemodeOverride
 global function GetSpawnpointGamemodeOverride
 global function AddSpawnpointValidationRule
+
+global function SetRespawnsEnabled
+global function RespawnsEnabled
 global function CreateNoSpawnArea
 global function DeleteNoSpawnArea
-
-global function FindSpawnPoint
+global function SpawnPointInNoSpawnArea
 
 global function RateSpawnpoints_Generic
 global function RateSpawnpoints_Frontline
-
-global function SetSpawnZoneRatingFunc
```

Comment:
Failing compile because `gamemodes/_gamemode_ctf.nut` is trying to use this global function (which is no longer global)

This is mentioned in https://github.com/R2Northstar/NorthstarMods/pull/830#issue-2445888371 but just putting it here as well because I didn't notice it until I looked at the other PR

Reply:
Oh good point on that! I believe with this entire rewriting i did, that function entirely lost its importance in the logic, i think i'll just remove it completely as it's not globalized anymore to be used by other scripts even. The same functionality this function would have can be set up directly by pointing a RatingFunc via `GameMode_SetPilotSpawnpointsRatingFunc` and `GameMode_SetTitanSpawnpointsRatingFunc` in the gamemode shared script for custom gamemodes, those are server-sided so it's safe that it wont break clients.

Reply:
Fixed with #861

Reply:
Alright #830 got merged so this got solved.